huxley/www/static/js/huxley/components/Huxley.js

```diff
@@ -18,11 +18,24 @@ var Huxley = React.createClass({
   mixins: [Router.Navigation],
 
   childContextTypes: {
-    session: React.PropTypes.number
+    session: React.PropTypes.number,
+    startDate: React.PropTypes.object,
+    endDate: React.PropTypes.object,
+    external: React.PropTypes.string,
+    registrationFee: React.PropTypes.number,
+    delegateFee: React.PropTypes.number
   },
 
   getChildContext: function() {
-    return {session: global.conferenceSession}
+    var conference = global.conference;
+    return {
+      session: conference['session'],
```

Comment:
perhaps we should just pass the conference down as context? we'd have to be careful not to mutate it anywhere, but it beats passing down every single field individually.